### ISSUE TITLE
[To Do] Add visual regression (Backstop) tests to Less

### ISSUE THREAD

**Issue by matthew-dean** [less/less.js#3262]
This came up for fixes to `svg-gradient()` (https://github.com/less/less.js/pull/3261). There are no automated visual regression tests, so it might be good to do so for functions that create visual assets. See: https://github.com/garris/BackstopJS

**Comment by Bruce-Hopkins** [comment]
Is this still relevent? Can I pick this up?

**Comment by khareyash05** [comment]
Is this open to work @matthew-dean ?

**Comment by matthew-dean** [comment]
Yep! I'm not sure if BackstopJS is still the best current solution, but yes I think it's still needed.

**Comment by KunalTiwari-git** [comment]
Hi! I'd like to contribute to this issue.

**Comment by Traveller-eng** [comment]
is it still open
